about.html

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Update this page?

Reply:
`about.html` was auto-generated when I ran `jekyll new myblog`, and is not linked anywhere. It was missed during development and has been deleted.